test/support/fixture_helper.rb

```diff
@@ -0,0 +1,133 @@
+# typed: ignore
+# frozen_string_literal: true
+
+require "rails_test_helper"
+
+module FixtureHelper
+  TEMP_FIXTURE_DIR = ROOT.join("tmp", "fixtures").to_s
+  FileUtils.mkdir_p(TEMP_FIXTURE_DIR)
+
+  Minitest.after_run do
+    # This does not get called on early exit, e.g. `Ctrl-C` or `(byebug) exit`.
+    FileUtils.remove_entry(TEMP_FIXTURE_DIR, true)
+  end
+
+  def around
+    old_working_dir = Dir.pwd
+    cache_rails_paths
+
+    yield
+
+    restore_rails_paths
+    Dir.chdir(old_working_dir)
+    FileUtils.remove_entry(@app_dir, true) if defined? @app_dir
+  end
+
+  def copy_template(template)
+    copy_dir("test/fixtures/#{template}")
+    Dir.chdir(app_dir)
```

Comment:
From this point on, the test uses `<your packwerk project dir>/tmp/fixtures/the_current_test` as the working directory.